draft-ietf-quic-tls.md

```diff
@@ -610,11 +612,11 @@ As TLS reports the availability of keying material, the packet protection keys
 and initialization vectors (IVs) are updated (see {{key-expansion}}).  The
 selection of AEAD function is also updated to match the AEAD negotiated by TLS.
 
-For packets other than any unprotected handshake packets (see {{cleartext-hs}}),
-once a change of keys has been made, packets with higher packet numbers MUST be
-sent with the new keying material.  The KEY_PHASE bit on these packets is
-inverted each time new keys are installed to signal the use of the new keys to
-the recipient (see {{key-phases}} for details).
+For packets other than any handshake packets (see {{hs-protection}}), once a
+change of keys has been made, packets with higher packet numbers MUST be sent
+with the new keying material.  The KEY_PHASE bit on these packets is inverted
```

Comment:
Why would one want to send more handshake packets without 1RTT keys once 1RTT keys have been established?

Reply:
Not sure what you mean.  This says that if you change keys, then you need to use the new keys and not the old ones.

Reply:
Nevermind, I'd been up since 3AM and ran out of brain.